src/test/java/teammates/test/driver/EmailAccount.java

```diff
@@ -26,32 +26,27 @@
      * @param username
      * @param password
      * @return registration key (null if cannot be found).
-     * @throws Exception
+     * @throws MessagingException
+     * @throws IOException
      */
     public static String getRegistrationKeyFromGmail(String username,
-            String password, String courseId) {
-        try {
-            Folder inbox = getGmailInbox(username, password);
-            Message[] messages = getMessages(inbox);
-            
-            // Loop over up to 5 unread messages at the top of the inbox
-            int maxEmailsToCheck = Math.min(messages.length, 5);
-            for (int i = messages.length - 1; i >= messages.length - maxEmailsToCheck; i--) {
-                Message message = messages[i];
-                
-                if (isRegistrationEmail(message, courseId)) {
-                    String body = getEmailBody(message);
-                    String key = getKey(body);
-                    message.setFlag(Flags.Flag.SEEN, true);
-                    
-                    inbox.close(true);
-                    return key;
-                }
+            String password, String courseId) throws IOException, MessagingException {
+        Folder inbox = getGmailInbox(username, password);
+        Message[] messages = getMessages(inbox);
+
+        // Loop over up to 5 unread messages at the top of the inbox
+        int maxEmailsToCheck = Math.min(messages.length, 5);
+        for (int i = messages.length - 1; i >= messages.length - maxEmailsToCheck; i--) {
+            Message message = messages[i];
+
+            if (isRegistrationEmail(message, courseId)) {
+                String body = getEmailBody(message);
+                String key = getKey(body);
+                message.setFlag(Flags.Flag.SEEN, true);
+
+                inbox.close(true);
+                return key;
```

Comment:
Got rid of the try-catch block, which erroneously "swallowed" crucial exceptions that might be thrown due to the security issues and what-nots. That reduces the indentation level of everything inside by 4 spaces. If you review with `?w=1` you'll notice that nothing inside the block is touched.